examples/android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">localhost</domain>
+        <domain includeSubdomains="true">10.0.2.2</domain>
+        <domain includeSubdomains="true">192.168.0.138</domain>
```

Comment:
Your local IP will change - perhaps just comment this out and leave a note for devs saying that they should add their local IP if they're testing the widget locally?